go/common/crypto/signature/signers/ledger/ledger_signer.go

```diff
@@ -0,0 +1,85 @@
+package ledger
+
+import (
+	"encoding/binary"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/oasislabs/ledger-go"
```

Comment:
Please group imports as mentioned [in the style guide](https://github.com/oasislabs/oasis-core/blob/master/CONTRIBUTING.md#go-style-guide).